README.md

```diff
@@ -1,2 +1,21 @@
 # eap-maven-plugin
 Maven Plugin to provision EAP 8
+
+# org.jboss.eap.plugins:eap-maven-plugin
+
+  The eap-maven-plugin is used to provision an EAP server that contains your application.
+
+## Goals Overview
+
+  General information about the goals.
+
+  * `eap:package` produces a fully configured EAP 8 server installation that contains your application. [package goal details](./doc/package.md) 
+
+  * `eap:help` displays plugin help information. Call `mvn eap:help -Ddetail=true -Dgoal=package` to get the list of configuration parameters and associated description.
```

Comment:
Should the line be:

`mvn org.jboss.eap.plugins:eap-maven-plugin:help -Ddetail=true -Dgoal=package`

So that the users can see how to configure the plugin before it is added to its pom.xml?